### PR TITLE
feat: feedback

### PR DESCRIPTION
**Pull request by catngh**
**What does this PR do?**

-`$feedback <optional_command> <description>`, the feedback will be sent to a feedback channel

https://user-images.githubusercontent.com/84314071/196651645-2b921f3b-58c3-45b2-8eb0-b2bf34016553.mov

### CODE REVIEW

File: src/commands/community/feedback/index.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                "Give feedback for the Mochi team about a specific command or in general",
+            }),
+          ],
+        },
+      }
+    }
+    let commandArg = args[1]
+    let feedback = ""
+    if (commandArg[0] == "$")
+      commandArg = commandArg.slice(1, commandArg.length).toUpperCase()
```
Comment (tuanddd) [inline src/commands/community/feedback/index.ts:57]:
```suggestion
    if (commandArg[0] === "$")
      commandArg = commandArg.slice(1).toUpperCase()
```

File: src/commands/community/feedback/index.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    else commandArg = ""
+
+    if (commandArg === "") feedback = args.slice(1, args.length).join(" ")
```
Comment (tuanddd) [inline src/commands/community/feedback/index.ts:60]:
```suggestion
    if (commandArg === "") feedback = args.slice(1).join(" ")
```

File: src/commands/community/feedback/index.ts [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    else feedback = args.slice(2, args.length).join(" ")
```
Comment (tuanddd) [inline src/commands/community/feedback/index.ts:61]:
```suggestion
    else feedback = args.slice(2).join(" ")
```

File: src/commands/community/feedback_slash/index.ts [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+      composeEmbedMessage(null, {
+        usage: `${SLASH_PREFIX}feedback <${SLASH_PREFIX}command> <description>`,
+        examples: `${SLASH_PREFIX}feedback i like it\n${SLASH_PREFIX}feedback $gm UI can be better`,
+        footer: [`Type ${SLASH_PREFIX}help for more actions!`],
+        description:
+          "Give feedback for the Mochi team about a specific command or in general",
+      }),
+    ],
+  }),
+  colorType: "Server",
```
Comment (tuanddd) [inline src/commands/community/feedback_slash/index.ts:75]: color type bị khác với text version

**Review by tuanddd — CHANGES_REQUESTED** [review]

**Review by nguyennh4522 — APPROVED** [review]

**Review by tuanddd — APPROVED** [review]

**Pull request merged**